eduiddashboard/views/__init__.py

```diff
@@ -57,7 +57,8 @@ class BaseFormView(FormView):
 
     def __init__(self, context, request):
         super(BaseFormView, self).__init__(request)
-        self.user = context.user
+        self.user = get_session_user(self.request)
+        retrieve_modified_ts(self.user, request.dashboard_userdb)
```

Comment:
That said, it was a think I almost objected to in the previous PR. I think there are a bunch of uses where we want to fetch the session user but without intent to end up saving a new user, so we could avoid calling retreive_modified_ts() and save a roundtrip to the db.